src/utils/utils.py

```
@@ -239,3 +227,82 @@ async def capture_screenshot(browser_context):
         return encoded
     except Exception as e:
         return None
+
+async def get_page_dom(browser_context):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_page_dom`

(ANN201)